.github/workflows/pathogen-repo-ci.yaml

```diff
@@ -56,7 +88,25 @@ permissions:
   packages: read
 
 jobs:
+  configuration:
+    runs-on: ubuntu-latest
+    steps:
+      - id: inputs
+        env:
+          runtimes: ${{ inputs.runtimes }}
+        shell: bash
+        run: |
+          runtimes="$(yq --output-format=json --indent=0 . <<<"$runtimes")"
+          echo runtimes="$runtimes" | tee -a "$GITHUB_OUTPUT"
+    outputs:
+      runtimes: ${{ steps.inputs.outputs.runtimes }}
+
   build:
+    needs: configuration
+    strategy:
+      matrix:
+        runtime: ${{ fromJSON(needs.configuration.outputs.runtimes) }}
```

Comment:
Thoughts on adding `fail-fast: false` so that runtime failures do not interfere with each other?

Reply:
Sounds like a good idea!

Reply:
Yep, that makes sense. Would you do it?